fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -96,20 +96,25 @@ private XMLGregorianCalendar journalDatoFrom(String jpDato) {
         }
     }
 
-    private Mottaker mottakerFrom(InfoRecord infoRecord) {
-        Organisasjon.Builder<Void> mottaker = Organisasjon.builder();
-        mottaker.withOrgnr(infoRecord.getIdentifier());
-        mottaker.withNavn(infoRecord.getOrganizationName());
+    private Adresse mottakerFrom(InfoRecord infoRecord) {
+        OrganisasjonDigitalAdresse.Builder<Void> orgAdr = OrganisasjonDigitalAdresse.builder();
+        orgAdr.withOrgnr(infoRecord.getIdentifier());
 
+        Adresse.Builder<Void> mottaker = Adresse.builder();
+        mottaker.withDigitalAdresse(orgAdr.build());
+
+//        mottaker.withNavn(infoRecord.getOrganizationName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)